js/dispatch_json.ts

```diff
@@ -30,7 +30,7 @@ function nextPromiseId(): number {
 }
 
 function decode(ui8: Uint8Array): JsonResponse {
-  const s = new TextDecoder().decode(ui8);
+  let s = new TextDecoder().decode(ui8);
```

Comment:
Now revert this to `const s` and this file goes untouched :)

Reply:
Doh, my bad, thanks again